python/function.py

```diff
@@ -2582,6 +2642,27 @@ def is_call_instruction(self, addr, arch=None):
 		return core.BNIsCallInstruction(self.handle, arch.handle, addr)
 
 	def set_user_var_value(self, var, def_addr, value):
+		"""
+		`set_user_var_value` allows the user to specify a PossibleValueSet value for a (MLIL) variable at its
```

Comment:
`value for an MLIL variable at its`...

instead of the parenthetical probably is better.